keywords/TestServerFactory.py

```diff
@@ -0,0 +1,50 @@
+from keywords.TestServerAndroid import TestServerAndroid
+from keywords.TestServeriOS import TestServeriOS
+from keywords.TestServerNetMono import TestServerNetMono
+from keywords.TestServerNetMsft import TestServerNetMsft
+
+
+class TestServerFactory:
+
+    @staticmethod
+    def validate_version_build(version_build):
+        if version_build is None:
+            raise ValueError("Make sure you provide a version / build!")
+
+        if len(version_build.split("-")) != 2:
+            raise ValueError("Make sure your version_build follows the format: 2.0.0-576")
+
+    @staticmethod
+    def validate_platform(platform):
+        valid_platforms = ["android", "ios", "net-mono", "net-msft"]
```

Comment:
you might want to add net-uwp, xamarin-ios and xamarin-android

Reply:
We should have a seperate PR for that as it needs lot of other changes and other files too.